python/tk_multi_publish2/api/manager.py

```diff
@@ -520,7 +520,7 @@ def _load_publish_plugins(self, context):
         engine = self._bundle.engine
 
         logger.debug(
-            "Finding publish plugin settings for context: %s" % (context,))
+            "Finding publish plugin settings (via pick_environment) for context: %s" % (context,))
```

Comment:
This is only true if the context is different than the current context. Should we instead have a separate debug log message in the else clause below?

Reply:
@thebeeland good catch. Updated to reflect when it happens.
How's this look?